docs: Add initial LMAT instructions (#8)
weaveworks-liquidmetal/liquid-metal-acceptance-tests: cmd/README.md

# Cmd

This is a small helper tool to run the Acceptance tests (LMATS) on remote
Equinix infrastructure.

This tool will change when we have built the scheduler component.

This tool will go away when I have figured out some more networking for the infra.

## What and why

There are 2 reasons it exists:
1. To save time on networking complexity during my initial stab at these tests,
  I chose not to set it up so that the CAPI management cluster could be run
  from outside the Equinix infra network.

>   _Technically_ then can be since the flintlock servers are bound to a public

Reviewer: Not sure about the then

```suggestion
  _Technically_ it could be since the flintlock servers are bound to a public
```

Author: think this might have been a `they` 🤔
